build-scripts/build-custom-app.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/bash
+
+# Prompt the user for the name of the new directory
+read -r -p "Enter the name of the new custom app (use camel-case if required): " custom_app_name
```

Comment:
Perhaps we need two prompt variables here. 

This one that currently exists should read: 
```suggestion
read -r -p "Enter the name of the new custom app website (use camel-case if required): " custom_app_name
```
This variable will be displayed on the homepage above the search bar.

We then want a second read variable that will hold the `theme` ([see below comment](https://github.com/climatepolicyradar/navigator-frontend/pull/205/files#r1724886782)).